From: Marc-André Lureau <marcandre.lureau@redhat.com>
Subject: controller: add missing spice/types include
Origin: upstream, http://cgit.freedesktop.org/spice/spice-protocol/commit/?id=2ffbca210033be262fc75e6a73742e0f0e6d5242
--- a/spice/controller_prot.h
+++ b/spice/controller_prot.h
@@ -18,6 +18,8 @@
 #ifndef _H_CONTROLLER_PROT
 #define _H_CONTROLLER_PROT
 
+#include <spice/types.h>
+
 #define CONTROLLER_MAGIC      (*(uint32_t*)"CTRL")
 #define CONTROLLER_VERSION    1
 
