From: Alon Levy <alevy@redhat.com>
Subject: qxl_dev: introduce defines for rings sizes
Origin: upstream, http://cgit.freedesktop.org/spice/spice-protocol/commit/?id=1a69e2bd757a41212166f748dcd4851265a27c0d

--- a/spice/qxl_dev.h
+++ b/spice/qxl_dev.h
@@ -200,10 +200,14 @@
     QXLPHYSICAL mem;
 } QXLSurfaceCreate;
 
-SPICE_RING_DECLARE(QXLCommandRing, QXLCommand, 32);
-SPICE_RING_DECLARE(QXLCursorRing, QXLCommand, 32);
+#define QXL_COMMAND_RING_SIZE 32
+#define QXL_CURSOR_RING_SIZE 32
+#define QXL_RELEASE_RING_SIZE 8
 
-SPICE_RING_DECLARE(QXLReleaseRing, uint64_t, 8);
+SPICE_RING_DECLARE(QXLCommandRing, QXLCommand, QXL_COMMAND_RING_SIZE);
+SPICE_RING_DECLARE(QXLCursorRing, QXLCommand, QXL_CURSOR_RING_SIZE);
+
+SPICE_RING_DECLARE(QXLReleaseRing, uint64_t, QXL_RELEASE_RING_SIZE);
 
 #define QXL_LOG_BUF_SIZE 4096
 
